rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/OperationEndpoint.java

```diff
@@ -154,7 +153,8 @@ DeliverableAnalyzerOperation updateDeliverableAnalyzer(
                             content = @Content(schema = @Schema(implementation = ErrorResponse.class))) })
     @GET
     @Path("/deliverable-analyzer/{id}")
-    @Consumes(MediaType.APPLICATION_JSON_PATCH_JSON) // workaround for PATCH support
+    @Consumes(MediaType.APPLICATION_JSON_PATCH_JSON)
+    // workaround for PATCH support
```

Comment:
Maybe put back the old formatting here? This reformatting makes less sense than the original